assets/sass/components/dashboard/_googlesitekit-consent-mode-setup-cta.scss

```diff
@@ -18,72 +18,39 @@
 
 .googlesitekit-plugin {
 
-	.googlesitekit-consent-mode-setup-cta-widget {
-
-		.googlesitekit-setup-cta-banner__primary-cell {
-			padding: $grid-gap-phone + 4px $grid-gap-phone 0;
-
-			@media (min-width: $width-tablet + px) {
-				padding: $grid-gap-desktop;
-			}
+	.googlesitekit-setup-cta-banner__svg-wrapper--consent-mode-setup-cta-widget {
```

Comment:
`--` should indicate a variant or state. Do we really need a special variant just for consent mode?